src/osp/Active/SysNewton.cpp

```diff
@@ -274,11 +274,10 @@ void SysNewton::create_body(ActiveScene& rScene, ActiveEnt entity,
         // Compute mass, center of mass
         Vector4 centerOfMass = compute_body_CoM(rScene, entHier.m_childFirst, Matrix4{});
         entBody.m_centerOfMassOffset = centerOfMass.xyz();
-        /* TODO: mass is left as the default (1.0f) for now; when we have the
-         * logic necesary to represent different thrusts per engine we can
-         * enable this; for now, real masses are too large.
-         */
-        //entBody.m_mass = centerOfMass.w();
+
+        // Compute moments of inertia
+        Matrix3 inertia = compute_body_inertia(rScene, entHier.m_childFirst,
+            centerOfMass.xyz(), Matrix4{});
```

Comment:
Default constructed Matrix4 instead of 0.0f?

Reply:
Default constructed Matrix4 should be an identity matrix, whereas {0.0f} will likely be all zeroes

Reply:
Hrmm, alright.